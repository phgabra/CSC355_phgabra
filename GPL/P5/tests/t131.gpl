int values[5];

int i = values[0];
double d = values[1];
string s = values[2];
