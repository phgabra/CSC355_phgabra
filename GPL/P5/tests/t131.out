gpl.cpp::main()
  input file(tests/t131.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
double d = 0
int i = 0
string s = "0"
int values[0] = 0
int values[1] = 0
int values[2] = 0
int values[3] = 0
int values[4] = 0
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
